src/lib.rs

```diff
@@ -34,13 +34,17 @@
 //! [`examples/bot`]: https://gitlab.com/whatisaphone/rlbot-rust/blob/master/examples/bot/main.rs
 //! [`examples/simple`]: https://gitlab.com/whatisaphone/rlbot-rust/blob/master/examples/simple.rs
 
+#![deny(warnings, missing_docs)]
+
+extern crate flatbuffers;
 extern crate libloading;
 extern crate ratelimit;
 
 pub use ffi::*;
 pub use framework::{run_bot, Bot};
 pub use packeteer::Packeteer;
 pub use rlbot::{init, RLBot};
+pub use rlbot_generated::rlbot::flat::*;
```

Comment:
After looking at the generated docs, I think I'd like to probably instead:

```
pub use rlbot_generated::rlbot::flat
```

Users can always import `rlbot::flat::*` if they want. I had done it this way to match the `ffi`, but I think we should do the same for `ffi`.

Reply:
Yeah the docs are basically unreadable now.

Reply:
```rust
#[doc(no_inline)]
pub use ffi::*;
pub use rlbot_generated::rlbot::flat;
#[doc(no_inline)]
pub use rlbot_generated::rlbot::flat::*;

pub mod ffi;
mod rlbot_generated;
```

This renders nicely. But I realize now that the two modules have overlapping names. I think we should pull them both out of the root module for now like you originally suggested.